pylistenbrainz/client.py

```diff
@@ -219,6 +219,27 @@ def submit_playing_now(self, listen):
         """
         return self._post_submit_listens([listen], LISTEN_TYPE_PLAYING_NOW)
 
+    def submit_feedback(self, feedback, recording_mbid):
```

Comment:
I think this function should be submit_user_feedback() -- that is the only thing that I see that we ought to change. Thanks!

Reply:
@mayhem done :)

Reply:
@mayhem done :)

Reply:
Thanks!

Reply:
@mayhem thx for merging :) is there a plan to make a v0.5.3 release tag  ?